boards/arm/b_g474e_dpow1/b_g474e_dpow1.dts

```diff
@@ -136,6 +162,8 @@
 	 */
 	psc-ucpdclk = <1>;
 	hbitclkdiv = <27>;
+	pinctrl-0 = <&ucpd1_cc1_pb6 &ucpd1_cc2_pb4>;
```

Comment:
It seems the west.yml commit which adds usbc pins support is gone.
Please put it back (prior to this commit) and since it is now merged , you can refer directly to the SHA1 of the hal_stm32 commit: c87358a3e9b99e0f8f18816f1a74777fe80d1ab6

Reply:
Done